Workbooks/ADXCluster/AtScale/Insights.workbook

```diff
@@ -572,11 +564,11 @@
                   },
                   {
                     "columnId": "microsoft.kusto/clusters-Cluster health-InstanceCount",
-                    "label": "Instance Count (Average)"
+                    "label": "Instance count (Average)"
                   },
                   {
                     "columnId": "microsoft.kusto/clusters-Cluster health-InstanceCount Timeline",
-                    "label": "Instance Count (Average) Timeline"
+                    "label": "Instance count (Avg) timeline"
```

Comment:
Is this change to fix the naming to match the metrics name? 
If not, then I suggest to be consistent, in some places you use Avg in others Average

Reply:
Thanks. This PR fixed other names inconsistencies. Changed the column name in all places to Avg